plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -1162,6 +1162,11 @@ private static function get_schema() {
 
 		$product_attributes_lookup_table_creation_sql = wc_get_container()->get( DataRegenerator::class )->get_table_creation_sql();
 
+		$feature_controller = wc_get_container()->get( FeaturesController::class );
+		$hpos_enabled =
```

Comment:
Not a request to change anything, more asking out of curiosity. At this stage, with HPOS on the verge of being marked stable, might we not simply create the HPOS tables as standard?

Reply:
Actually, that's not a bad idea. I think we can do in preparation for a larger move. Would you like to create an issue for this?

Reply:
[Created](https://github.com/woocommerce/woocommerce/issues/39826) (and added to the project board) ✅